--- ../orig/openssh-7.1p1/authfile.c	2015-08-21 14:49:03.000000000 +1000
+++ ./authfile.c	2015-09-24 15:29:31.484800000 +1000
@@ -180,6 +180,10 @@
 #ifdef HAVE_CYGWIN
 	if (check_ntsec(filename))
 #endif
+        /*
+        For running OpenSSH binaries outside of Cygwin, skip the
+        permissions check, because Windows permissions can't be
+        reliably mapped to POSIX permissions.
 	if ((st.st_uid == getuid()) && (st.st_mode & 077) != 0) {
 		error("@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@");
 		error("@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @");
@@ -190,6 +194,7 @@
 		error("This private key will be ignored.");
 		return SSH_ERR_KEY_BAD_PERMISSIONS;
 	}
+        */
 	return 0;
 }
 
