--- authfile.c.orig	2017-04-12 07:47:11.064400000 +1000
+++ authfile.c	2017-04-07 20:42:50.778000000 +1000
@@ -178,9 +178,13 @@
 	 * permissions of the file. if the key owned by a different user,
 	 * then we don't care.
 	 */
+         /*
 #ifdef HAVE_CYGWIN
 	if (check_ntsec(filename))
 #endif
+        For running OpenSSH binaries outside of Cygwin, skip the
+        permissions check, because Windows permissions can't be
+        reliably mapped to POSIX permissions.
 	if ((st.st_uid == getuid()) && (st.st_mode & 077) != 0) {
 		error("@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@");
 		error("@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @");
@@ -191,6 +195,7 @@
 		error("This private key will be ignored.");
 		return SSH_ERR_KEY_BAD_PERMISSIONS;
 	}
+	*/
 	return 0;
 }
 
